public/app/features/logs/components/LogRows.tsx

```diff
@@ -53,6 +54,11 @@ export interface Props extends Themeable2 {
   isFilterLabelActive?: (key: string, value: string) => Promise<boolean>;
   pinnedRowId?: string;
   containerRendered?: boolean;
+  /**
+   * If true, the `contain:strict` css property will be added to the wrapping `<table>` for performance reasons.
+   * Any overflowing content will be clipped at the table boundary.
+   */
+  containContent?: boolean;
```

Comment:
Wouldn't it be easier to flip the logic? Otherwise you would need to add the prop also to other places where `LogRows` is used.

Reply:
Good point, should be fixed now!